packages/@react-aria/tooltip/src/useTooltip.ts

```diff
@@ -18,6 +18,9 @@ interface TooltipAria {
   tooltipProps: HTMLAttributes<HTMLElement>
 }
 
+/**
+ * Handles accessibility for a Tooltip.
```

Comment:
I think we usually say "Provides" instead of "Handles"?

```suggestion
 * Provides the accessibility implementation for a Tooltip component.
```